public/index.html

```diff
@@ -127,5 +127,30 @@ <h2>CSS loading animations with minimal effort!</h2>
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <canvas hidden width="64" height="64" id="favicon-canvas"></canvas>
+    <div style="display:none;">
+      <img id="favicon-source" src="/favicon.ico" width="64" height="64"/>
+    </div>
+    <script>
+    var canvas = document.getElementById("favicon-canvas");
```

Comment:
Does this still work if we create the canvas off-render? As in not in the DOM?

Could we instead create a canvas with `document.createElement('canvas')` and never append it to the DOM?